### ISSUE TITLE
Visual label 'Refresh' is not part of accessibility name(Programmatic name) for "Refresh" button :A11y_DocumentDB for VS Code Extension_Add a New Document_Devtools

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode-documentdb#374]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #FTP; #Win32; #DocumentDB for VS Code Extension; #SH_DocumentDB_Win32_NOV2025; #WCAG2.5.3; #Devtools; 

## Environment Details:
Application Name: DocumentDB for VS Code Extension
Version 0.6.0
OS: Windows 11[Version 23H2(OS Build 22631.3593)]  

## Repro Steps:

1. Open VS code 
2. Install the DocumentDB for VS Code Extension and open it
3. Navigate to the DocumentDB Connections area.
4. Click the New Connection action.
5. Select Connection String as the connection method.
6. Paste the connection string when prompted.
7. Choose to authenticate with username/password and enter the credentials
8. Confirm to create the connection.
9. Locate the desired cluster in the DocumentDB Connections list.
10. Expand the cluster entry to initiate the connection.
11. Expand one of the databases under the cluster.
12. Expand a collection within that database.
13. Select the Documents item for that collection.
14. Collection View will open
15. TAB till add new document button and press ENTER key
16. TAB till Refresh  button
17. Observe the issues

# Actual Result: 
Visual label 'Refresh' is not part of accessibility name(Programmatic name) for "Refresh" button
**Note:** Same issue repro with Validate button

## Expected Result:
The accessible name must include the visible text label as an unbroken string.

## User Impact:
If the visible label (e.g., "Refresh") does not match or is not part of the programmatic name, assistive technology users may not understand the purpose of the button or may be unable to activate it using voice commands.

## Attachment

<img width="1114" height="718" alt="Image" src="https://github.com/user-attachments/assets/4e801203-4dcc-4788-b08a-d867c838fb69" />

Similar issue

<img width="1102" height="713" alt="Image" src="https://github.com/user-attachments/assets/57bbbedc-ce82-4b87-86cf-349a8eb562da" />
